### PR TITLE
Fix all Warnings for GCC 9

### PR DESCRIPTION
**Pull request by thestr4ng3r**
@abcSup Can you elaborate whether the `vtop()` function should be used anywhere? It is currently dead code.

### CODE REVIEW

File: libr/anal/p/anal_ppc_cs.c [hunk 1]
```diff
@@ -67,6 +67,7 @@ static const char* cmask32(const char *mb_c, const char *me_c) {
 	return cmask;
 }
 
+#if 0
```
Comment (radare) [inline libr/anal/p/anal_ppc_cs.c]: dont if0, just rm

File: libr/anal/p/anal_ppc_cs.c [hunk 2]
```diff
@@ -223,7 +225,9 @@ static void opex(RStrBuf *buf, csh handle, cs_insn *insn) {
 #define PPCSPR(n) getspr(&gop, n)
 #define ARG(n) getarg2(&gop, n, "")
 #define ARG2(n,m) getarg2(&gop, n, m)
+#if 0
```
Comment (radare) [inline libr/anal/p/anal_ppc_cs.c]: same

File: libr/bin/format/dmp/dmp64.c [hunk 3]
```diff
@@ -67,6 +67,7 @@ static int r_bin_dmp64_init_header(struct r_bin_dmp64_obj_t *obj) {
 	return true;
 }
 
+#if 0
```
Comment (radare) [inline libr/bin/format/dmp/dmp64.c]: same

File: libr/include/r_vector.h [hunk 4]
```diff
@@ -69,6 +69,9 @@ R_API void r_vector_free(RVector *vec);
 R_API RVector *r_vector_clone(RVector *vec);
 
 static inline bool r_vector_empty(const RVector *vec) {
+	if (!vec) {
```
Comment (radare) [inline libr/include/r_vector.h]: assert

**Comment by abcSup** [comment]
> @abcSup Can you elaborate whether the `vtop()` function should be used anywhere? It is currently dead code.

It is currently dead code and was meant to use for vaddr to paddr address translation later on.

File: libr/bin/format/dmp/dmp64.c (hunk 3 above)
Comment (XVilka) [reply libr/bin/format/dmp/dmp64.c]: No, not in this case. As far I understand this code might be useful in the future, for implementing more features: https://github.com/radareorg/radare2/issues/16048

**Comment by radare** [comment]
ping @thestr4ng3r

**Comment by thestr4ng3r** [comment]
I've removed the dead code. In case it will be needed later it can be grabbed from the git history.

**Pull request merged**